Makefile

```
@@ -7,7 +7,7 @@ COVERAGE=coverage.out
 
 MCLI_SOURCE_FILES?=./cmd/mongocli
 MCLI_BINARY_NAME=mongocli
-MCLI_VERSION?=$(shell git describe --always --tags)
+MCLI_VERSION?=$(shell  git describe --always --tags | cut -d "/" -f 2)
```

I think we should still use `v` here. We don’t want the `v` in the version
```suggestion
MCLI_VERSION?=$(shell  git describe --always --tags | cut -d "v" -f 2)
```